docs/proposals/Release-Supply-Chain-Metadata.md

```diff
@@ -0,0 +1,174 @@
+  MediaType:   application/vnd.oci.image.manifest.v1+json
+  Platform:    unknown/unknown
+  Annotations: 
+    vnd.docker.reference.digest: sha256:f1b520af44d5e22b9b8702cbbcd651092df8672ed7822851266b17947c2a0962
+    vnd.docker.reference.type:   attestation-manifest
+               
+  Name:        ghcr.io/ratify-project/ratify:v1.3.0@sha256:c936d0ed115975ee7fc8196fbc5baff8100e92bff3d401c60df6396b9451e773
+  MediaType:   application/vnd.oci.image.manifest.v1+json
+  Platform:    unknown/unknown
+  Annotations: 
+    vnd.docker.reference.type:   attestation-manifest
+    vnd.docker.reference.digest: sha256:6105d973c1c672379abfdb63486a0327d612c4fe67bb62e4d20cb910c0008aa9
+```
+
+You can see the `attestation-manifest` reference-type.
+
+To inspect, it's recommended to use docker's image inspection tool:
+
+```shell
+docker buildx imagetools inspect ghcr.io/ratify-project/ratify:v1.3.0 --format '{{ json .SBOM }}'
+```
+
+### Referrer Artifacts
+
+Ratify should support generating SBOMs via an OSS tool and attach to the published container images via ORAS. This would be in line with the Ratify verification capabilities supported by the project.
+
+### SBOM Binary Artifacts
+
+Ratify should support publishing an SBOM in a common format like SPDX as a release artifact published next to each OS/arch specific binary.
+
+GoRelease already support automatic SBOM generation and Ratify should update GoReleaser to take advantage of this support. Example [here](https://github.com/goreleaser/goreleaser-example-supply-chain/blob/d6d60f6320dbe97bda24b6351d9afa2035b3a23a/.goreleaser.yaml#L48).
+
+## Provenance
+
+### Docker Buildx Attestation
+
+Please refer to SBOM section for format [details](#docker-buildx-attestations).
+
+Ratify also added SLSA provenance attestation generation support as part of docker buildx. Similar to SBOM support, this adds a SLSA provenance attestation to the image index during image build.
+
+### Referrer Artifacts (TBD)
+
+As a future improvement, Ratify can look into attaching the SLSA build provenance metadata as a referrer artifact attached to the image. This might be in the form of a standalone artifact or packaged in an in-toto attestation.
+
+### Provenance Binary Artifacts
+
+Ratify should support publishing an provenance file for each OS/arch binary. The OSSF scorecard also recommends (and looks for) a provenance file in the release.
+
+This would involve adding an extra workflow to generate the SLSA provenance for each output binary after GoReleaser has finished. Example [here](https://github.com/slsa-framework/slsa-github-generator/blob/main/.github/workflows/generator_generic_slsa3.yml).
+
+## Proposed Dev Implementation
+
+### Open Questions
+
+1. Is it ok to use a self-signed certificate for Ratify's signing purposes? Yes, we are ok with this.
+2. How do we handle certificate revocation scenarios? Is it Ratify's responsibility to resign all the release and dev images?
```

Comment:
We can follow our support policy, and resign latest release ONLY.

Reply:
Ok sure that works